src/OrchardCore/OrchardCore.Contents.Core/ContentSearchOption.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+
+namespace OrchardCore.Contents;
+
+public class ContentSearchOption
```

Comment:
If we keep the options pattern, maybe `ContentsAdminListFilterOptions` ;)

Or `ContentsAdminListOptions`, at least `ContentsSearchOptions`, I don't know.

Reply:
done